### ISSUE TITLE
Adding utterance app for comments on the github page website of my workflowr

### ISSUE THREAD

**Issue by ogorodriguez** [workflowr/workflowr#245]
Is there a way to add comments on our github pages website using the utterances app? (https://utteranc.es/)

**Comment by ogorodriguez** [comment]
I have seen this that seems to work but I don't know how to set it for pages on the workflowr repo.
https://github.com/utterance/utterances/issues/503

**Comment by jdblischak** [comment]
Yes, it should be possible to include [utterances](https://utteranc.es/) comments on your workflowr pages by including the HTML snippet they provide:

```
<script src="https://utteranc.es/client.js"
        repo="[ENTER REPO HERE]"
        issue-term="pathname"
        theme="github-light"
        crossorigin="anonymous"
        async>
</script>
```

You can do this via the R Markdown strategy I described in https://github.com/jdblischak/workflowr/issues/240#issuecomment-792719266. The main problem is that this doesn't seem to work any more, at least not for me. Could you please test it out and report back if you can successfully include an HTML file across all of your website pages? I want to confirm that someone else has this bug before I investigate further.
